packages/block-editor/src/components/block-patterns-list/README.md

```diff
@@ -18,10 +18,10 @@ For more infos about blocks patterns, read [this](https://make.wordpress.org/cor
 Renders a block patterns list.
 
 ```jsx
-import { BlockPatternList } from '@wordpress/block-editor';
+import { BlockPatternsList } from '@wordpress/block-editor';
```

Comment:
Since this component is a public export from the `@wordpress/block-editor` package, I don't think we can easily change its exported name. Probably safer to keep the previous name

Reply:
We definitely can't change it like this. If we really wanted to rename it, we should add a deprecation and fallback to the renamed one. In general though I think the correct naming is the existing one(using singular), no?

Reply:
> Since this component is a public export from the @wordpress/block-editor package, I don't think we can easily change its exported name.

This specific change doesn't actually alter any exports or public APIs, it just updates the documentation to match current usage.

> In general though I think the correct naming is the existing one(using singular), no?

[The file name is `block-patterns-list`](https://github.com/WordPress/gutenberg/blob/272d0a9c61551ab2da8d682896e6baec0fdcb93d/packages/block-editor/src/components/block-patterns-list/index.js) (plural), and [almost everywhere imports it as such](https://github.com/search?q=repo%3AWordPress%2Fgutenberg%20%22__experimentalBlockPatternsList%20as%20BlockPatternsList%22&type=code):
```js
import { __experimentalBlockPatternsList as BlockPatternsList } from '@wordpress/block-editor';
```

[The only place I've found that doesn't](https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%22BlockPatternList%22+AND+%22%2Fblock-patterns-list%22&type=code) is [the `block-patterns-tab` component](https://github.com/WordPress/gutenberg/blob/272d0a9c61551ab2da8d682896e6baec0fdcb93d/packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js#L25), which [I also updated (see below)](https://github.com/WordPress/gutenberg/pull/56210/files#diff-60c3ea8d0743a260802cc7cd7d7c114fea7521d4b19a3143e18de9dac035e772):
```js
import BlockPatternList from '../../block-patterns-list';
```

[The public export is here](https://github.com/WordPress/gutenberg/blob/272d0a9c61551ab2da8d682896e6baec0fdcb93d/packages/block-editor/src/components/index.js#L158), and that isn't changing:
```js
export { default as __experimentalBlockPatternsList } from './block-patterns-list';
```

[The component itself is exported as a default](https://github.com/WordPress/gutenberg/blob/272d0a9c61551ab2da8d682896e6baec0fdcb93d/packages/block-editor/src/components/block-patterns-list/index.js#L188), so while I did rename the function, it shouldn't make any difference to consumers, internal or external:
```js
export default forwardRef( BlockPatternList );
```

So ultimately this doesn't affect much, just tidies things a bit and establishes better internal consistency.

With all that said though, if we'd prefer to leave things as they are, I'm happy to revert these particular changes.

Reply:
>[The public export is here](https://github.com/WordPress/gutenberg/blob/272d0a9c61551ab2da8d682896e6baec0fdcb93d/packages/block-editor/src/components/index.js#L158), and that isn't changing:

Didn't check that, sorry. You're right that it doesn't affect the public API. Regarding the singular/plural I don't have any strong opinion. Since we use mostly the plural, your change seems fine to me.